### DISCUSSION TITLE
Get an error when trying to send a reset user password on supabase admin UI - "Failed to reset user password: failed to make recover request: Unable to process request"

### DISCUSSION THREAD

**Discussion by codeteacher330** [supabase/supabase#18953] [Questions]
Hi, I get an error when trying to send a reset user password on supabase admin UI - "Failed to reset user password: failed to make recover request: Unable to process request". Also same fails on maginc link sending and invitation sending.
I am attaching the image. please fix
![resentpassworderror](https://github.com/supabase/supabase/assets/127133513/fce79546-4ee8-4a5e-829c-6cd8a6c9ad53)

**Comment by Lurrobert** [comment]
Got the same error right now
Have no clue what happened

**Reply by tteguayco** [reply]
Getting this error too...

**Comment by ChainCoderX** [comment]
+1
